README.md

```diff
@@ -12,61 +13,62 @@ It is being developed in the context of the Google Summer of Code 2017 program,
 
 Please read the project's [Wiki page](https://github.com/mushorg/go-dpi/wiki) for more information.
 
+For documentation, please check out the [godoc reference](https://godoc.org/github.com/mushorg/go-dpi).
+
 ## Example usage
 
 The library and the modules APIs aim to be very simple and straightforward to use. The library relies on the [gopacket](https://godoc.org/github.com/google/gopacket) library and its Packet structure. Once you have a Packet in your hands, it's very easy to classify it with the library.
-First you need a flow that contains the packet. There is a helper function for constructing a flow from a single packet. Simply call:
+First of all you need to initialize the library. You can do that with simply:
```

Comment:
Alright :) I'll push these changes to README.md to my other PR, as that's where that commit belongs, and rebase this PR once the other one is merged.